### ISSUE TITLE
Deprecation warnings in 22.0.1

### ISSUE THREAD

**Issue by fjellvannet** [jaraco/keyring#492]
Hello,
I use version 22.0.1 running pip on python 3.8.5, all updated recently. Unfortunately the current version has gotten deprecation warnings from cryptography.utils using set_password and get_password:
```
/.../venv/lib/python3.8/site-packages/secretstorage/dhcrypto.py:16: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/.../venv/lib/python3.8/site-packages/secretstorage/util.py:25: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
```
Looks as if this could be fixed easily by just replacing the now deprecated functions in your code with new ones. 
Wish you a nice day :)

**Comment by mitya57** [comment]
These warnings are coming from secretstorage, not from keyring. Already fixed in SecretStorage v3.3.1 via mitya57/secretstorage#27.

**Comment by fjellvannet** [comment]
You're right that fixed it. Thx :)

tor. 11. feb. 2021, 17:54 skrev Dmitry Shachnev <notifications@github.com>:
